docs/build-instructions.md

```diff
@@ -81,3 +81,18 @@ When installing, you must minimally select all Visual C++ components.
     - `./pingviewer`
 
 > **Tip**: You can use `make -jN` where N is the number of threads in your computer to improve the compiling speed.
+
+## Build documentation
+
+1. Follow the **Source Code** guide.
```

Comment:
What does this line mean, exactly?

Reply:
Here: https://github.com/bluerobotics/ping-viewer/pull/970/files#diff-c5bc07b84a84f251d50ddb55eec5c5c2645500ded34a5f36e31fcc162c97d803R9

Reply:
The link is pointing nowere.

Reply:
Check the line 9 of this same file.

Reply:
Oh, ok.
Maybe "Follow the **Source Code** guide to get the source files." would be more clear.